packages/env/lib/env.js

```diff
@@ -390,6 +390,14 @@ async function configureWordPress( environment, config ) {
 			options
 		);
 	}
+
+	for ( const [ key, value ] of Object.entries( config.config ) ) {
+		await dockerCompose.run(
+			'cli',
```

Comment:
Need to pass `tests-cli` if `environment === 'tests'`.

Reply:
Oops